index.html

```diff
@@ -29,13 +29,105 @@
         <![endif]-->
     <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
       <div class="container">
-        <div class="navbar-header">
+        <!-- <div class="navbar-header">
           <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
             <span class="sr-only">Toggle navigation</span>
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
-          </button>
+          </button> -->
+          <ul class = "nav nav-tabs">
+            <li role="presentation" class="active"><a href="#">Home</a></li>
+              <li role="presentation"><a href="#">About</a></li>
+              <li class="dropdown">
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Find a Poem <span class="caret"></span></a>
```

Comment:
This should be all replaced with something less hardcoded. I would recommend a plugin such as this one: http://mmenu.frebsite.nl/. We had discussed making this menus and submenus contain dynamic content that is based off of either counts from the API we used or popularity of the site users. This plugin contains the ability to dynamically add menu options and has a mobile first format to it. Overall, I think this code is too clunky to be maintained. This should be used _only_ as a proof of concept.